python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -364,24 +365,52 @@ class RandomForestClassifier(Base):
             raise ValueError("Wrong value passed in for max_features"
                              " please read the documentation")
 
-    def _get_model_info(self):
+    def _convert_to_treelite(self):
+        task_category = self.num_classes
```

Comment:
task_category should essentially be num_classes, because when we convert rf to treelite the function in c++ will use the task_category value for classification as the num_classes. We were using a default value of 2 since we only use FIL for binary class classification.
https://github.com/rapidsai/cuml/blob/1b6f14183b660cf29229f321845515ef0b9d7fcd/cpp/src/randomforest/randomforest.cu#L298